_includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md

```diff
@@ -75,7 +76,7 @@ Make sure that the store and locale are set correctly.
 namespace Pyz\Glue\GlueBackendApiApplication;
 
 use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
-use Spryker\Glue\StoresRestApi\Plugin\Application\StoreHttpHeaderApplicationPlugin;
+use Spryker\Glue\StoresBackendApi\Plugin\Application\StoreApplicationPlugin;
```

Comment:
Please fix namespaces, to match the code in the PR after refactoring. (Same for SAPI)